### ISSUE TITLE
PropertySex (VixenX) matching but not populating metadata

### ISSUE THREAD

**Issue by covrdinbeez** [PAhelper/PhoenixAdult.bundle#1007]
**Describe the bug**
Manual search/matching is working through Plex, but when the proper match is selected, the title is just being populated with "One Condition [Propertysex/Property Sex] 2018-04-11"

**Include**
1. Scene URL
https://www.propertysex.com/scene/4354300/one-condition

2. File Name
PropertySex Avi Love One Condition 720p HEVC x265-piemonster.mkv

2a. Search string
PropertySex One Condition

3. Log File
```
2020-09-05 17:11:18,222 (700006753000) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM2CjcxODU1OXMyCmlkczI1ClByb3BlcnR5U2V4IE9uZSBDb25kaXRpb25zNApuYW1lbnM0CnllYXJyMAo_
2020-09-05 17:11:18,222 (700006753000) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2020-09-05 17:11:18,223 (700006753000) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM2CjcxODU1OXMyCmlkczI1ClByb3BlcnR5U2V4IE9uZSBDb25kaXRpb25zNApuYW1lbnM0CnllYXJyMAo_
2020-09-05 17:11:18,223 (700006753000) :  INFO (agentkit:961) - Searching for matches for {'year': None, 'id': '718559', 'name': 'PropertySex One Condition'}
2020-09-05 17:11:18,224 (700006753000) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/718559/tree'
2020-09-05 17:11:18,227 (700006753000) :  INFO (__init__:881) - *******MEDIA TITLE****** PropertySex One Condition
2020-09-05 17:11:18,227 (700006753000) :  INFO (__init__:881) - Getting Search Settings for: PropertySex One Condition
2020-09-05 17:11:18,227 (700006753000) :  INFO (PAsearchSites:1189) - mediaTitle w/ possible abbreviation: PropertySex One Condition
2020-09-05 17:11:18,232 (700006753000) :  INFO (PAsearchSites:1452) - mediaTitle w/ possible abbrieviation fixed: PropertySex One Condition
2020-09-05 17:11:18,234 (700006753000) :  INFO (PAsearchSites:1141) - Site found with method #3
2020-09-05 17:11:18,234 (700006753000) :  INFO (PAsearchSites:1452) - ^^^^^^^ siteID: 733
2020-09-05 17:11:18,234 (700006753000) :  INFO (PAsearchSites:1452) - ^^^^^^^ Shortening Title
2020-09-05 17:11:18,234 (700006753000) :  INFO (PAsearchSites:1452) - propertysex  vs property sex
2020-09-05 17:11:18,234 (700006753000) :  INFO (PAsearchSites:1430) - 3
2020-09-05 17:11:18,234 (700006753000) :  INFO (PAsearchSites:1459) - 8
2020-09-05 17:11:18,234 (700006753000) :  INFO (PAsearchSites:1471) - searchTitle (before date processing): One Condition
2020-09-05 17:11:18,235 (700006753000) :  INFO (__init__:881) - Search Title: One Condition
2020-09-05 17:11:18,235 (700006753000) :  INFO (__init__:881) - One%20Condition
2020-09-05 17:11:18,236 (700006753000) :  INFO (PAutils:105) - Requesting HEAD "https://www.propertysex.com"
2020-09-05 17:11:18,605 (700006753000) :  INFO (PAutils:105) - Requesting GET "https://site-api.project1service.com/v2/releases?type=scene&search=One%20Condition"
2020-09-05 17:11:19,091 (700006753000) :  INFO (PAutils:105) - Requesting GET "https://site-api.project1service.com/v2/releases?type=movie&search=One%20Condition"
2020-09-05 17:11:19,445 (700006753000) :  INFO (PAutils:105) - Requesting GET "https://site-api.project1service.com/v2/releases?type=serie&search=One%20Condition"
2020-09-05 17:11:19,787 (700006753000) :  INFO (PAutils:105) - Requesting GET "https://site-api.project1service.com/v2/releases?type=trailer&search=One%20Condition"
2020-09-05 17:11:20,131 (700006753000) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2020-09-05 17:11:20,131 (700006753000) :  DEBUG (runtime:924) - Response: [200] str, 4208 bytes
2020-09-05 17:11:22,825 (700006753000) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgo3MTg1NTlzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzNTkKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdDovLzQzNTQzMDB8NzMzfHNjZW5lP2xhbmc9ZW5zNApndWlkczE3CjQzNTQzMDB8NzMzfHNjZW5lczIKaWRyMAo_
2020-09-05 17:11:22,825 (700006753000) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2020-09-05 17:11:22,826 (700006753000) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgo3MTg1NTlzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzNTkKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdDovLzQzNTQzMDB8NzMzfHNjZW5lP2xhbmc9ZW5zNApndWlkczE3CjQzNTQzMDB8NzMzfHNjZW5lczIKaWRyMAo_
2020-09-05 17:11:22,826 (700006753000) :  DEBUG (model:32) - Loading model with GUID com.plexapp.agents.phoenixadult://4354300|733|scene?lang=en
2020-09-05 17:11:22,827 (700006753000) :  DEBUG (model:234) - Deserializing from /Users/batcave/Library/Application Support/Plex Media Server/Metadata/Movies/8/861cc635187dc1d5730675da322575b45b885db.bundle/Contents/com.plexapp.agents.phoenixadult/Info.xml
2020-09-05 17:11:22,828 (700006753000) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/718559/tree'
2020-09-05 17:11:22,831 (700006753000) :  INFO (__init__:893) - ******UPDATE CALLED*******
2020-09-05 17:11:22,831 (700006753000) :  INFO (__init__:896) - 733
2020-09-05 17:11:22,832 (700006753000) :  CRITICAL (agentkit:1095) - Exception in the update function of agent named 'PhoenixAdult', called with guid 'com.plexapp.agents.phoenixadult://4354300|733|scene?lang=en' (most recent call last):
  File "/Applications/Plex Media Server.app/Contents/Resources/Plug-ins-a78fef9a9/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/api/agentkit.py", line 1093, in _update
    agent.update(obj, media, lang, **kwargs)
  File "/Users/batcave/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/__init__.py", line 1524, in update
    metadata = PAsearchSites.siteAussieAss.update(metadata, siteID, movieGenres, movieActors)
  File "/Users/batcave/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/network1service.py", line 84, in update
    token = get_Token(siteNum)
NameError: global name 'siteNum' is not defined

2020-09-05 17:11:22,832 (700006753000) :  DEBUG (model:229) - Serializing to /Users/batcave/Library/Application Support/Plex Media Server/Metadata/Movies/8/861cc635187dc1d5730675da322575b45b885db.bundle/Contents/com.plexapp.agents.phoenixadult/Info.xml
2020-09-05 17:11:22,833 (700006753000) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2020-09-05 17:11:22,834 (700006753000) :  DEBUG (runtime:924) - Response: [200] str, 16 bytes
```

**Comment by covrdinbeez** [comment]
Didn't realize when I posted this that these were coming from the same metadata pull as bug #1006. I made the same change as PowerModder suggested in his post, and it fixed my metadata populating issue as well.

**Comment by DirtyRacer1337** [comment]
Fixed in aafe942177574d1804a79d447f76f786c72d86be
